### ISSUE TITLE
Display of BYTES from CockroachDB wrong?

### ISSUE THREAD

**Issue by Kukulkano** [beekeeper-studio/beekeeper-studio#611]
Hi. A CockroachDB table was defined like this:

```
CREATE TABLE dv.data (
  VID BYTES NOT NULL,
  PAYLOAD BYTES NOT NULL,
  PROVIDERID SMALLINT NOT NULL,
  CREATIONDATE TIMESTAMPTZ NOT NULL,
  PRIMARY KEY (VID)
);
```

I insert this:

`INSERT INTO dv.data (VID, PAYLOAD, PROVIDERID, CREATIONDATE) VALUES ('1dac139bf2017020d74cced2ce520b78', 'abcdefg', 1, NOW());`

Now select in CockroachDB shell:

```
> select vid from dv.data where vid='1dac139bf2017020d74cced2ce520b78';
                vid
------------------------------------
  1dac139bf2017020d74cced2ce520b78
(1 row)
```
That is fine. But, with the same query, BeeKeeper is showing me the VID content like this:

MWRhYzEzOWJmMjAxNzAyMGQ3NGNjZWQyY2U1MjBiNzg

**Comment by rathboma** [comment]
Hey! It''s almost certainly how I'm translating the field for display. I'm just calling `toString`.

I'll poke around to see if it can be fixed, but we can't usually assume that a `bytes` field contains a valid string

**Comment by Kukulkano** [comment]
Yes, I understand. In my case it is UTF-8 encoded. Some others may do UTF-16/Unicode. Also binary. Maybe you do an option/setting to decide about how to handle binary in general?

Show binary data...
O as UTF-8 text
O as Unicode text
O as HEX values

In my case I would select the first option.

**Comment by rathboma** [comment]
I think this is a great candidate for a settings page.

Tagging with #197.
